### ISSUE TITLE
Make Show Private persistent

### ISSUE THREAD

**Issue by tristanlatr** [twisted/pydoctor#492]
I think when users are showing the private API, it should persist across all pages. To do this we can use cookie, like I did to show/hide the sidebar: https://github.com/twisted/pydoctor/blob/b957492b0eef69f7fce1e6f0b362137f478f1e1b/pydoctor/themes/base/sidebartoggle.js

The question is: what should be the cookie time to to live value? 24h looks fine to me.

**Comment by tristanlatr** [comment]
Re-clicking on show private on every page is just not possible. Especially if a lot of stuff is markqued as private be cause we defer the privacy handling to official Sphinx docs.

**Comment by not-my-profile** [comment]
Yes ... my point is that you shouldn't be using private APIs. If an API ought to be public, it ought to be documented in Sphinx for projects that use Sphinx. So imho the proper fix is to send a PR to add the missing documentation instead of making it easier to lookup private APIs.

**Comment by tristanlatr** [comment]
Ok, i get your point. Know that it’s been a long time that I what’s this feature to be implemented. It’s not only because of  pydoc.dev. But I would be much easier to navigate the docs with this feature. Sending PR to all project covered by pydoc.dev seems impossible.
